presto-hive-hadoop2/conf/docker-compose.yml

```diff
@@ -5,9 +5,10 @@ services:
     image: '${HADOOP_BASE_IMAGE}:${DOCKER_IMAGES_VERSION}'
     ports:
       - '1180:1180'
-      - '8020:8020'
+      - '8020:8020' # Default hadoop namenode port
       - '8042:8042'
       - '8088:8088'
+      - '9000:9000' # Default hadoop namenode port
```

Comment:
we now have two ports described as "Default hadoop namenode port". Can you elaborate what is this for?